runconfig/hostconfig_windows.go

```diff
@@ -10,25 +10,22 @@ import (
 // DefaultDaemonNetworkMode returns the default network stack the daemon should
 // use.
 func DefaultDaemonNetworkMode() container.NetworkMode {
-	return container.NetworkMode("default")
+	return container.NetworkMode("NAT")
```

Comment:
Perhaps a unit test to verify that --netmode=nat performs the same as --netmode=NAT? Minor comment though.